popup/popup.js

```diff
@@ -1,15 +1,16 @@
 // Polyfill for browser compatibility
-if (typeof browser === 'undefined') {
-    var browser = chrome;
-}
+if (typeof browser === "undefined") globalThis.browser = chrome;
 
-const months = new Map([['jan', 1], ['feb', 2], ['mar', 3], ['apr', 4], ['may', 5], ['jun', 6], ['jul', 7], ['aug', 8], ['sep', 9], ['oct', 10], ['nov', 11], ['dec', 12]])
+const months = new Map([[1, 'jan'], [2, 'feb'], [3, 'mar'], [4, 'apr'], [5, 'may'], [6, 'jun'], [7, 'jul'], [8, 'aug'], [9, 'sep'], [10, 'oct'], [11, 'nov'], [12, 'dec']]);

```

Comment:
Correct, I forgot to revert this one. It would be necessary for displaying the meta as `oct 2024` in the popup messages - reverting for the time being.